src/EditorFeatures/CSharp/EmbeddedLanguages/CSharpEmbeddedLanguageEditorFeaturesProvider.cs

```diff
@@ -24,7 +22,7 @@ public CSharpEmbeddedLanguageEditorFeaturesProvider()
         {
         }
 
-        internal override string EscapeText(string text, SyntaxToken token)
+        public override string EscapeText(string text, SyntaxToken token)
```

Comment:
not a fan of internal members for us when not necessary.  'public' aligns with teh majority case on how we generally code.